bentoml/frameworks/coreml.py

```diff
@@ -81,6 +81,7 @@ def pack(self, model):  # pylint:disable=arguments-differ
                 "CoreMLModelArtifact can only pack type 'coremltools.models.MLModel'"
             )
 
+        super().pack(model, metadata=metadata)
```

Comment:
would it make more sense to just add these `super` calls in the `__getattribute__` method in BentoServiceArtifact?